Assets/HoloToolkit-Examples/UX/Scripts/InteractiveGroup.cs

```diff
@@ -58,7 +59,10 @@ void CreateInteractives()
             {
                 GameObject PrefabInst = Instantiate(InteractivePrefab, gameObject.transform) as GameObject;
                 InteractiveToggle InterInst = PrefabInst.GetComponent<InteractiveToggle>();
-                Interactives.Add(InterInst);
+                if (InterInst != null)
+                {
+                    Interactives.Add(InterInst);
+                }
```

Comment:
Should we throw a warning if our prefab doesn't have the `InteractiveToggle` telling the dev to add one to their prefab?

Reply:
Yes.